src/engine.rs

```diff
@@ -372,28 +365,21 @@ where
             LogItemReader::new_file_reader(file_builder, path)?
         };
 
-        Ok(DumpIterator {
-            raft_grous_ids: raft_groups,
-            item_reader,
-        })
+        Ok(DumpIterator { item_reader })
     }
 }
 
-pub struct DumpIterator<'a, B: FileBuilder> {
-    raft_grous_ids: &'a [u64],
+pub struct DumpIterator<B: FileBuilder> {
```

Comment:
You can directly implement it for `LogItemReader` now (in file_pipe_log::debug).

Reply:
OK